qadence/backends/pulser/pulses.py

```diff
@@ -75,8 +75,9 @@ def add_pulses(
         if block.qubit_support.is_global:
             (uuid, duration) = block.parameters.uuid_param("duration")
             t = evaluate(duration) if duration.is_number else sequence.declare_variable(uuid)
-            pulse = Pulse.ConstantPulse(duration=t, amplitude=0, detuning=0, phase=0)
-            sequence.add(pulse, GLOBAL_CHANNEL, "wait-for-all")
+            # pulse = Pulse.ConstantPulse(duration=t, amplitude=0, detuning=0, phase=0)
```

Comment:
Remove ?

Reply:
Ah thanks for the reminder. I meant to double-check this with @nmheim and @vytautas-a actually.